src/main/java/com/coherent/training/selenium/stanila/base/BaseClass.java

```diff
@@ -1,31 +1,34 @@
 package com.coherent.training.selenium.stanila.base;
-
-import com.coherent.training.selenium.stanila.pages.LoginPOM;
+import com.coherent.training.selenium.stanila.pages.*;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.asserts.SoftAssert;
-import java.time.Duration;
-
 public class BaseClass {
     public static WebDriver driver;
     public LoginPOM loginPOM;
+    public MultiSelectPOM multiSelectPOM;
+    public JsAlertBoxPOM jsAlertBoxPOM;
+    public DynDataLoadingPOM dynamicDataLoadingPOM;
+    public ProgressBarPOM progressBarPOM;
+    public WebTablePOM webTablePOM;
```

Comment:
declare and init the poms where they needed

Reply:
Implemented